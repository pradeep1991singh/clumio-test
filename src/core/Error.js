import React from "react";

const Error = ({ message }) => {
  return (
    <div className="block dark:bg-gray-900 dark:text-gray-500 p-10">
      {message}
    </div>
  );
};

export default Error;
