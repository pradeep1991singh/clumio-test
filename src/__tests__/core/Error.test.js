import React from "react";
import { render } from "@testing-library/react";
import Error from "../../core/Error";

describe("<Error />", () => {
  test("Snapshot", () => {
    const { asFragment } = render(<Error message="errored" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
