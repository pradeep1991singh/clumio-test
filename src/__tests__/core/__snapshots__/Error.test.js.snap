// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> Snapshot 1`] = `
<DocumentFragment>
  <div
    class="block dark:bg-gray-900 dark:text-gray-500 p-10"
  >
    errored
  </div>
</DocumentFragment>
`;
